Keas.Mvc/Controllers/Api/PeopleController.cs

```diff
@@ -174,6 +174,20 @@ public async Task<IActionResult> SearchUsers(string searchTerm)
                 User = user,
                 Title = await _identityService.GetTitle(user.Iam)
             };
+            var supervisor = await _identityService.GetIamSupervisor(user.Iam);
+            if (supervisor != null)
+            {
+                var superPerson = await _context.People
+                    .Where(x => x.Team.Slug == Team && x.UserId == supervisor.Id)
+                    .Include(x => x.Supervisor)
+                    .IgnoreQueryFilters()
+                    .FirstOrDefaultAsync();
+                if (superPerson != null)
+                {
+                    person.SupervisorId = superPerson.Id;
```

Comment:
I'm not sure it's a problem to set both but best practice is just to set the ID when doing these FK relationships